src/cpp/session/resources/schema/user-prefs-schema.json

```diff
@@ -187,6 +187,11 @@
                 "console_right_on_top": {
                     "type": "boolean",
                     "description": "Whether the console is on top when it is on the right side of the workbench."
+                },
+                "additional_source_columns": {
```

Comment:
Putting the option in this object will cause it to get reset if you use an older version of RStudio (i.e. if you make any layout changes in 1.3 then they will destroy your 1.4 layout as 1.3 will write out an object that doesn't include `additional_source_columns`). This is probably acceptable but if you need to work around it you could create a new pref that has layout for 1.4 only.